src/components/FormSelectInput.component.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import styled from "styled-components";
+
+import { defaultTheme } from "../theme";
+import { InputStyle } from "./styled";
+
+type MapElement = { id: number; name: string };
+interface State extends MapElement {
+  cities: Array<MapElement>;
+}
+export const FormSelectInput = ({
+  label,
+  register,
+  onFocus,
+  onBlur,
+  onChange,
+  loading,
+  data,
+  disabled,
+}: {
+  label: string;
+  register: any;
+  onFocus?: any;
+  onBlur?: any;
+  onChange?: (e: React.ChangeEvent) => void;
```

Comment:
please configure your linter to keep optional parameters and the end, also I think is better to use an interface here, avoide using `any` , have you use the spread operator to pass props before? I think we saw one example in our syncs

Reply:
Didn't find the eslint rule to do that, but everything else has been refactored as of now! Thanks